app/javascript/shared/components/EmojiPicker.res

```diff
@@ -0,0 +1,90 @@
+open Webapi.Dom
+
+type emojiEvent = {
+  id: string,
+  native: string,
+  unifield: string,
+  shortcodes: string,
+}
+
+type args = {
+  title: string,
+  ref: React.ref<Js.Nullable.t<Dom.element>>,
+  theme: string,
+  onEmojiSelect: emojiEvent => unit,
+  data: Js.Json.t,
+}
+
+@module("emoji-mart") @new external picker: args => unit = "Picker"
+
+type data
+@module external data: Js.Json.t = "@emoji-mart/data"
+
+let emojiDivClassName = isOpen => {
+  switch isOpen {
+  | true => "absolute top-11 sm:top-full sm:left-full z-10 shadow left-1/2 -translate-x-1/2 sm:translate-x-0"
+  | false => "hidden"
+  }
```

Comment:
The emojiPicker goes offscreen in **Ipad** and creates horizontal overflow to its container (review section)

![image](https://user-images.githubusercontent.com/26390669/176430280-e68d56e9-3934-4a00-b980-aeb5d1c13270.png)